bleak/backends/bluezdbus/client.py

```diff
@@ -904,6 +916,27 @@ async def write_gatt_descriptor(
             "Write Descriptor %s | %s: %s", descriptor.handle, descriptor.obj[0], data
         )
 
+    def _read_notify_fd(self, fd, callback):
+        loop = asyncio.get_running_loop()
+        os.set_blocking(fd, False)
+
+        def on_data():
+            try:
+                data = os.read(fd, 1024)
+                if not data:  # EOF, close file descriptor
+                    os.close(fd)
+                    return
+                callback(bytes(data))
+            except Exception as e:
```

Comment:
We don't catch exceptions from notification callbacks anywhere else. We just let it crash the program since it is considered a bug in the program. So I would take that out of try/except here.